persistence/pg.go

```diff
@@ -355,6 +351,40 @@ func (p *PostgresPersister) settleHTLC(ctx context.Context,
 	return nil
 }
 
+func (p *PostgresPersister) changeInvoiceStatus(ctx context.Context,
```

Comment:
should it be in a separate commit? (ideally it should have been in the refactoring PR..)